trino/trino_test.go

```diff
@@ -330,83 +330,83 @@ func TestTypeConversion(t *testing.T) {
 		t.Fatal(err)
 	}
 	testcases := []struct {
-		PrestoType                       string
-		PrestoResponseUnmarshalledSample interface{}
-		ExpectedGoValue                  interface{}
+		TrinoType                       string
```

Comment:
How about `s/TrinoType/dataType`? These doesn't have to be exported and the prefix is unnecessary